[IOS] FIX: Return original filename instead of FullSizeRender (#990)
fluttercandies/flutter_photo_manager: CHANGELOG.md

# CHANGELOG

> ## 2.7.3

Reviewer: We didn't publish 2.7.2 yet, so these should be under the 2.7.2 section.

### Fixes

> - [iOS] Add PHAsset extension method `getUntouchedResource` to retrieve original media instead of one with adjustments / filters.
> - [iOS] Fix: When retrieving an asset title, now returns original file name instead of `FullSizeRender.*` if this has adjustments. (#976)

Reviewer: ```suggestion
- Retrieve original media instead of one with adjustments/filters for subtype files on iOS. (#976)
- Returns original file name instead of `FullSizeRender.*` if this has adjustments on iOS. (#976)
```
